package.json

```diff
@@ -100,11 +101,11 @@
     "circleci:test": "node node_modules/mocha/bin/mocha --max-old-space-size=4096 --harmony --trace-deprecation test/*.test.js test/*.unittest.js",
     "circleci:lint": "npm run lint-files",
     "build:examples": "cd examples && node buildAll.js",
-    "pretest": "npm run lint-files",
+    "pretest": "npm run setup && npm run lint-files",
     "lint-files": "npm run lint && npm run schema-lint",
-    "lint": "eslint lib bin hot buildin \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\"",
+    "lint": "eslint setup lib bin hot buildin \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\"",
     "fix": "npm run lint -- --fix",
-    "pretty-files": "prettier \"lib/**.*\" \"bin/**.*\" \"hot/**.*\" \"buildin/**.*\" \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\" --write",
+    "pretty-files": "prettier \"setup/**.*\" \"lib/**.*\" \"bin/**.*\" \"hot/**.*\" \"buildin/**.*\" \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\" --write",
```

Comment:
Please change to `setup/*.js`, that one is wrong, others one are fixed in https://github.com/webpack/webpack/pull/6864/files

Reply:
done